Gems/ScriptCanvas/Code/scriptcanvasgem_files.cmake

```diff
@@ -0,0 +1,545 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(FILES
+    Source/SystemComponent.cpp
+    Source/ScriptCanvasCommonGem.cpp
+    Source/PerformanceStatistician.cpp
+    Source/PerformanceTracker.cpp
+    Include/ScriptCanvas/Asset/AssetRegistry.cpp
+    Include/ScriptCanvas/Asset/ExecutionLogAsset.cpp
+    Include/ScriptCanvas/Asset/RuntimeAsset.cpp
+    Include/ScriptCanvas/Asset/RuntimeAssetHandler.cpp
+    Include/ScriptCanvas/Asset/RuntimeInputs.cpp
+    Include/ScriptCanvas/Asset/SubgraphInterfaceAsset.cpp
+    Include/ScriptCanvas/Asset/SubgraphInterfaceAssetHandler.cpp
+    Include/ScriptCanvas/Core/ExecutionNotificationsBus.cpp
+    Include/ScriptCanvas/Core/Connection.cpp
+    Include/ScriptCanvas/Core/Contract.cpp
+    Include/ScriptCanvas/Core/Contracts/MethodOverloadContract.cpp
+    Include/ScriptCanvas/Core/Contracts/RestrictedNodeContract.cpp
+    Include/ScriptCanvas/Core/Core.cpp
+    Include/ScriptCanvas/Core/Datum.cpp
+    Include/ScriptCanvas/Core/EBusHandler.cpp
+    Include/ScriptCanvas/Core/Endpoint.cpp
+    Include/ScriptCanvas/Core/Graph.cpp
+    Include/ScriptCanvas/Core/GraphData.cpp
+    Include/ScriptCanvas/Core/GraphSerialization.cpp
+    Include/ScriptCanvas/Core/MethodConfiguration.cpp
+    Include/ScriptCanvas/Core/ModifiableDatumView.cpp
+    Include/ScriptCanvas/Core/Node.cpp
+    Include/ScriptCanvas/Core/Nodeable.cpp
+    Include/ScriptCanvas/Core/NodeableNode.cpp
+    Include/ScriptCanvas/Core/NodeableNodeOverloaded.cpp
```

Comment:
Are all of these .cpp files supposed to be in an Include/ directory? Seems a bit odd...

Reply:
It is, one of my next PR's will address this